components/Common/Dropdown/index.module.scss

```diff
@@ -28,3 +27,16 @@
     }
   }
 }
+.dropdownList::-webkit-scrollbar {
+  width: var(--space-06);
+}
+
+.dropdownList::-webkit-scrollbar-track {
+  background: transparent;
+}
+
+.dropdownList::-webkit-scrollbar-thumb {
+  background-color: rgba(155, 155, 155, 0.7);
```

Comment:
Can we use a colour that we already have on our `tokens.scss`? And then apply an rgba out of that colour?